content/concepts/nat/autonat.md

```diff
@@ -10,17 +10,26 @@ aliases:
 ## Background
 
 While the [identify protocol][spec_identify] lets peers inform each other about their observed network
-addresses, however, it is possible that some of these addresses are not accessible from outside the network, as the peer may be located in a private network that is behind a [NAT](overview.md) and therefore unreachable.
+addresses, however, it is possible that some of these addresses are not accessible from outside the network,
+as the peer may be located in a private network that is behind a [NAT](overview.md) and therefore unreachable.
```

Comment:
```suggestion
as the peer may be located in a private network that (behind a NAT or a firewall).
```

Reply:
Also add: Advertising addresses that are not reachable is detrimental for the health of a p2p network, as other nodes will unsuccessfully try to dial those addresses.